src/utils/dummy_generator.py

```diff
@@ -0,0 +1,38 @@
+from steamship import PluginInstance, Steamship, Tag, Task
+from steamship.agents.schema import AgentContext
+
+from generators.image_generator import ImageGenerator
+from generators.music_generator import MusicGenerator
+from schema.objects import Item
+
+
+class DummyGenerator(MusicGenerator, ImageGenerator):
+    plugin_instance: PluginInstance
+
+    def __init__(self, client: Steamship):
+        super().__init__(plugin_instance=client.use_plugin("test-image-generator"))
+
+    def dummy_generation(self) -> Task:
+        return self.plugin_instance.generate(text="Dummy", tags=[Tag(kind="dummy")])
```

Comment:
question (non-blocking): is having a call to `generate()` useful?  Or could this be a no-op that just returns a fully-formed Task ?